src/leetCode.ts

```diff
@@ -0,0 +1,129 @@
+import { Request, Response } from 'express';
+import * as gqlQueries from './GQLQueries';
+import * as formatUtils from './FormatUtils';
+import * as controllers from './Controllers';
```

Comment:
By using index.ts files that export everything from their folders contents I was able to simplify your import statements, its also possible to destructure the imports but I like this style more today!